lib/src/tinycolor.dart

```diff
@@ -160,6 +172,6 @@ class TinyColor {
   bool equals(Object other) => this == other;
 }
 
-extension _ on Color {
+extension ColorExtension on Color {
```

Comment:
I think the name should start with `_`. It is not for public?

Reply:
True. Please add `_`